usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -282,8 +217,16 @@ def fiscal_years_for_processing(options: list) -> list:
     return [int(x) for x in options["fiscal_years"]]
 
 
-def check_new_index_name_is_ok(provided_name: str) -> None:
-    if not provided_name.endswith(settings.ES_TRANSACTIONS_NAME_SUFFIX):
-        raise SystemExit(
-            "new index name doesn't end with the expected pattern: '{}'".format(settings.ES_TRANSACTIONS_NAME_SUFFIX)
-        )
+def check_new_index_name_is_ok(provided_name: str, type: str) -> None:
```

Comment:
Cleaner to provide the suffix name to the function instead of type and having to choose which suffix to use